cfe_internal/update/update_policy.cf

```diff
@@ -139,13 +139,14 @@ bundle agent cfe_internal_setup_python_symlink(symlink_path)
         move_obstructions => "true",
         if => isvariable("python");
 
-      "$(sys.bindir)/python" -> { "CFE-3512" }
+      "$(sys.bindir)/python" -> { "CFE-3512", "CFE-4146" }
         delete => u_tidy,
-        if => islink( "$(sys.bindir)/python" ),
+        if => and( islink( "$(sys.bindir)/python" ),
+                   strcmp( "$(sys.bindir)", "/var/cfengine/bin")),
         comment => concat( "We don't want to leave a python that is potentially in $PATH ",
                            "after having re-named our python symlink that is used for various ",
-                           "modules");
-
+                           "modules. Additionally we want to be cautious that we don't delete ",
+                           "system python symlinks in the event the binary was built for FHS.");
```

Comment:
I know what FHS is, but wonder if we don't need a glossary somewhere or just replace `FHS` with `https://wiki.linuxfoundation.org/lsb/fhs`.

Reply:
We have https://docs.cfengine.com/docs/3.21/resources-faq-fhs.html

You think we should add the term here? https://docs.cfengine.com/docs/3.21/resources-additional-topics-glossary.html#top